### DISCUSSION TITLE
Stop calculation button

### DISCUSSION THREAD

**Discussion by yanisr** [cables-gl/cables_docs#452] [Feature Requests] [answered]
I would be great to have a stop or pause button or shortcut to save computer resources.

**Comment by steam0r** [comment, marked as answer]
pressing ctrl/cmd-p and selecting these might be what you want?

![Bildschirm­foto 2023-01-09 um 09 38 46](https://user-images.githubusercontent.com/151271/211268162-15a0342b-4548-4b04-b142-b960d358bfa6.png)

if you want this "patched" you can use "active" in mainloop:

![Bildschirm­foto 2023-01-09 um 09 36 37](https://user-images.githubusercontent.com/151271/211267839-dcfa3bab-2784-4c7f-834c-16129cf9a5e3.png)

and if you somehow want to control this in code or from "the outside" this would be the way to go:

![Bildschirm­foto 2023-01-09 um 09 36 17](https://user-images.githubusercontent.com/151271/211267934-9551b038-3439-40e6-af87-b7bfe2e657a2.png)
